plugins/inputs/traefik/README.md

```diff
@@ -0,0 +1,87 @@
+# Telegraf Input Plugin: Traefik
+
+This plugin gather health check status from services registered in Traefik. Traefik must be started with the `--api` option (formerly `--web`). See the [configuration docs](https://docs.traefik.io/configuration/api/) for more information.
+
+### Configuration:
+
+Simple configuration, with one measurement per request capture:
+
+```toml
+# Description
+[[inputs.traefik]]
+  ## Required Traefik server address, host and port (default: "127.0.0.1")
+  # address = "http://127.0.0.1:8080"
+  #
+  # [inputs.traefik.tags]
+  #  instance = "prod"
```

Comment:
Remove the `tags` usage example, we don't usually show it even though it can, of course, be set on all plugins.  Just showing the default config will be enough.